lib/container/Circular_Buffer.fz

```diff
@@ -86,6 +86,12 @@ is
   => abstract
 
 
+  # discard all unread elements
```

Comment:
Maybe drop this and instead user has to create a new instance of a circular buffer?